### ISSUE TITLE
Markup data type not exported

### ISSUE THREAD

**Issue by clojj** [jtdaugherty/brick#213]
Hi, 
is there a reason, why the constructor of Markup is not exported ?
like `Markup(..)`
I would like to use it for pattern matching.

https://github.com/jtdaugherty/brick/blob/master/src/Data/Text/Markup.hs#L9

I need to test if some Markup is empty like this:
```
            Just m@(Markup ls) ->
                if null ls then txt $ T.unlines $ getEditContents e else markup m
```
So, a function to do this test would also be Ok !

**Comment by jtdaugherty** [comment]
It's not exported because it's not intended to be part of the public API since the internals could change, but I just added `empty`.

**Comment by clojj** [comment]
I totally agree, the empty function is the best solution, thx

**Comment by jtdaugherty** [comment]
This is released in `0.45`.

**Comment by jtdaugherty** [comment]
FYI, this will be renamed to `isEmpty` in the next release.
